generators/app/index.js

```diff
@@ -1,125 +1,122 @@
+    this.fs.copy(this.templatePath('editorconfig.txt'), this.destinationPath('.editorconfig'));
+    this.fs.copy(this.templatePath('gitignore.txt'), this.destinationPath('.gitignore'));
+    this.fs.copy(this.templatePath('data.json'), this.destinationPath('data.json'));
+    this.fs.copy(this.templatePath('gulpfile.babel.js'), this.destinationPath('gulpfile.babel.js'));
+    this.fs.copy(this.templatePath('.babelrc'), this.destinationPath('.babelrc'));
+    this.fs.copy(this.templatePath('readme.md'), this.destinationPath('readme.md'));
+    this.fs.copy(this.templatePath('LICENSE'), this.destinationPath('LICENSE'));
 
-    this.fs.copy(getTemplate('.htaccess'), this.destinationPath('.htaccess'));
-    this.fs.copy(getTemplate('editorconfig.txt'), this.destinationPath('.editorconfig'));
-    this.fs.copy(getTemplate('data.json'), this.destinationPath('data.json'));
-    this.fs.copy(getTemplate('gulpfile.js'), this.destinationPath('gulpfile.js'));
-    this.fs.copy(getTemplate('readme.md'), this.destinationPath('readme.md'));
-    this.fs.copy(getTemplate('js/data-factory.js'), this.destinationPath('src/factories/'+properties.prefixedName+'data/'+properties.prefixedName+'data.factory.js'));
+    this.fs.copy(this.templatePath('js/modernizr.js'), this.destinationPath('src/js/modernizr.js'));
+    this.fs.copy(this.templatePath('scss/**/*'), this.destinationPath('src/scss'));
+    this.fs.copy(this.templatePath('view/view.html'), this.destinationPath('src/views/home/home.html'));
+    this.fs.copy(this.templatePath('view/view.js'), this.destinationPath('src/views/home/home.js'));
+    this.fs.copy(this.templatePath('view/view.scss'), this.destinationPath('src/views/home/home.scss'));
 
-    this.fs.copy(getTemplate('gitignore.txt'), this.destinationPath('.gitignore'));
+    this.fs.copy(this.templatePath('js/data-factory.js'), this.destinationPath(`src/factories/${props.prefix}-data/${props.prefix}-data.factory.js`));
 
     this.fs.copyTpl(
-      getTemplate('index.html'),
+      this.templatePath('index.html'),
       this.destinationPath('index.html'),
-      properties
+      props
     );
     this.fs.copyTpl(
-      getTemplate('js/app.js'),
+      this.templatePath('js/app.js'),
       this.destinationPath('src/js/app.js'),
-      properties
+      props
     );
     this.fs.copyTpl(
-      getTemplate('package.json'),
+      this.templatePath('package.json'),
       this.destinationPath('package.json'),
-      properties
+      props
     );
   }
 
+  // Install Queue
   install () {
-    var gen = this;
     this.installDependencies({
-            npm: false,
-            bower: false,
-            yarn: true,
-      callback: function () {
-        gen.log(chalk.green('Your app is ready. Build something awesome!'));
-        gen.spawnCommand('gulp', ['vendors']);
-        gen.spawnCommand('gulp');
+      npm: false,
+      bower: false,
+      yarn: true,
+      callback: () => {
+        this.spawnCommandSync('gulp', ['build']);
```

Comment:
I changed this to just the build task because I think in most cases you don't want to launch right into the watcher (and you may want to edit the gulpfile with a proxied URL in which case the standalone browser sync isn't what you want anyway).